modules/node_modules/@ncigdc/containers/FrequentlyMutatedGenesTable.js

```diff
@@ -145,6 +146,11 @@ const FrequentlyMutatedGenesTableComponent = compose(
             ),
           },
           {
+            key: 'annotations',
+            title: 'Annotations',
+            style: { padding: '5px 0 0 0', textAlign: 'center' },
```

Comment:
sorry, didn't notice this before. I think this padding should be applied to a wrapper around the icons, this style is applied to the header as well so it makes it not line up with the rest :)